esm-samples/jsapi-angular-cli/src/app/app.component.ts

```diff
@@ -70,31 +70,29 @@ export class AppComponent implements OnInit, OnDestroy {
     return this.view.when();
   }
 
-  // tslint:disable-next-line:use-lifecycle-interface
   ngOnInit(): any {
 
     // Set this property when using routes in order to resolve the /assets correctly.
     // IMPORTANT: the directory path may be different between your product app and your dev app
     // config.assetsPath = "/assets";
-    config.assetsPath = "assets/"
+    config.assetsPath = 'assets/';
 
     this.zone.runOutsideAngular(() => {
       // Initialize MapView and return an instance of MapView
-      this.initializeMap().then((mapView) => {
+      this.initializeMap().then(() => {
         // The map has been initialized
         this.zone.run(() => {
           console.log('mapView ready: ');
-        })
+        });
       });
 
     });
   }
 
-  // tslint:disable-next-line:use-lifecycle-interface
```

Comment:
Class `AppComponent` implements interface `OnDestroy` - removed this comment as it does not have any effect.